### PR TITLE
Add wdl_runner pipeline example.

### PR DESCRIPTION
**Pull request by mbookman**

### CODE REVIEW

**Comment by deflaux** [comment]
LGTM

File: wdl_runner/README.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+virtual machines.
+First a master node is created for Cromwell, and then Cromwell submits
+each stage of the workflow as one or more separate pipelines.
+
+Execution of a running Pipeline proceeds as:
+
+1. Create Compute Engine virtual machine
+
+2. Execute wdl_runner.py
+    a. Run Cromwell (server)
```
Comment (pgrosu) [inline wdl_runner/README.md]: Might need a space between these two lines as they appear on the same line after being reformatted.

File: wdl_runner/cromwell_launcher/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+RUN apt-get update && \
+    apt-get install python --yes && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install gcloud
+# See https://cloud.google.com/sdk/
+RUN curl https://sdk.cloud.google.com | bash
+
+# Add the install location explicity to the path (for non-interactive shells)
```
Comment (pgrosu) [inline wdl_runner/cromwell_launcher/Dockerfile]: s/explicity/explicitly

File: wdl_runner/cromwell_launcher/setup.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+apt-get clean && \
+rm -rf /var/lib/apt/lists/*
+
+# Install gcloud
+# See https://cloud.google.com/sdk/
+if [[ ! -e /root/google-cloud-sdk ]]; then
+  curl https://sdk.cloud.google.com | bash
+fi
+
+# Add the install location explicity to the path (for non-interactive shells)
```
Comment (pgrosu) [inline wdl_runner/cromwell_launcher/setup.sh]: s/explicity/explicitly

**Comment by pgrosu** [comment]
Hi Matt,

I really like it!  Just a few minor things.  

How long does the pipeline take to complete once it is started?

Thanks,
Paul

**Comment by mbookman** [comment]
Thanks for the feedback Paul.

The sample pipeline takes about 20 minutes to complete.

**Comment by pgrosu** [comment]
You're very welcome Matt, and many thanks for helping the community!  Thank you for the completion time, and like you said there might be alternate approaches but at least this works for now :)

**Pull request merged**